src/main/java/spleetwaise/transaction/logic/commands/EditCommand.java

```diff
@@ -41,19 +41,26 @@ public class EditCommand extends Command {
      * The message usage string that explains how to use this command.
      */
     public static final String MESSAGE_USAGE =
-            COMMAND_WORD + ": Edit a transaction.\n" + "Parameters: " + " INDEX (must be a positive integer) "
-                    + PREFIX_PHONE + "CONTACT " + PREFIX_AMOUNT
-                    + "AMOUNT " + PREFIX_DESCRIPTION + "DESCRIPTION " + "[" + PREFIX_DATE + "DATE ]" + PREFIX_CATEGORY
-                    + "FOOD\n" + "Example: " + COMMAND_WORD + " " + PREFIX_PHONE + "88888888 " + PREFIX_AMOUNT
-                    + "10.00 " + PREFIX_DESCRIPTION + "Paid John for lunch" + PREFIX_DATE + "23012024 "
-                    + PREFIX_CATEGORY + "FOOD";
+            COMMAND_WORD + ": Edit a transaction.\n"
+                    + "Parameters: INDEX (must be a positive integer) "
+                    + "[" + PREFIX_PHONE + "CONTACT] "
```

Comment:
The phone field isn't needed, right?

Reply:
hmm why not?

Reply:
when you say "isn't needed" do you mean it's optional? or that `editTxn` should not support editing the person field of a Transaction?

Reply:
Yeah I am not sure if  `editTxn` should support editing the person field of a Transaction.

Reply:
I didn't realise it was supported.

Reply:
As it is currently supported, it should be reflected in the UG

Reply:
Agreed